lib/transaction_search.rb

```diff
@@ -144,7 +145,8 @@ def handle_csv_search
   end
 
   def email_csv_export
-    AccountTransactionReportMailer.delay.csv_report_email(to_email, @order_details.pluck(:id), @date_range_field)
+    order_detail_ids = @order_details.respond_to?(:pluck) ? @order_details.pluck(:id) : @order_details.map(&:id)
```

Comment:
We went with this to give us the performance/memory boost if it's a relation, but then still work if it's already an array.

Part of me wanted to monkey patch `Array` to support `pluck`, but that seemed like a bad idea.